.github/workflows/test.yml

```diff
@@ -19,21 +19,25 @@ jobs:
     container:
       image: sagemath/sagemath:latest
       options: --user root
-
     steps:
-    - name: Install git on container
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Install git
       run: |
         apt update
         apt install -y git
 
-    - name: Checkout
-      uses: actions/checkout@v3
-      with:
-        submodules: true
-
+      # TODO Point to tag draft-irtf-cfrg-vdaf-11 when it's available.
     - name: Install dependencies
-      run: sage -pip install git+https://github.com/cfrg/draft-irtf-cfrg-vdaf@72227878#subdirectory=poc
+      run: |
+        sage -pip install mypy
+        sage -pip install git+https://github.com/cfrg/draft-irtf-cfrg-vdaf@72227878#subdirectory=poc
```

Comment:
TODO: Update once https://github.com/cfrg/draft-irtf-cfrg-vdaf/pull/383 is merged.